cmd/ipfs/daemon.go

```diff
@@ -76,6 +80,17 @@ the port as you would other services or database (firewall, authenticated proxy,
 	Run:         daemonFunc,
 }
 
+// defaultMux tells mux to serve path using the default muxer. This is
+// mostly useful to hook up things that register in the default muxer,
+// and don't provide a convenient http.Handler entry point, such as
+// expvar and http/pprof.
+func defaultMux(path string) func(node *core.IpfsNode, mux *http.ServeMux) (*http.ServeMux, error) {
+	return func(node *core.IpfsNode, mux *http.ServeMux) (*http.ServeMux, error) {
+		mux.Handle(path, http.DefaultServeMux)
+		return mux, nil
+	}
+}
+
```

Comment:
They're just handlers for `/debug/vars` and `/debug/pprof`, nothing happens unless somebody sends a HTTP request for those. I'm not sure I understand the questions.